style_login.css

```diff
@@ -46,13 +46,40 @@ body {
 .login_box {
 	background: #ccb0f7;
 	box-sizing: border-box;
-	border-radius: 5%;
 	position: relative;
-	grid-column: 2;
+	grid-column: 1;
 	grid-row: 1;
 	display: grid;
 }
 
+/* Style for text input */
+input[type=text] {

```

Comment:
Below is where I added style for the text input field,  password input field and submit input button.